Trappist/src/Promact.Trappist.Repository/Questions/QuestionRepository.cs

```diff
@@ -32,15 +33,16 @@ public ICollection<SingleMultipleAnswerQuestionApplicationClass> GetAllQuestions
         /// </summary>
```

Comment:
please update the description to a method to add single multiple answer question.